src/copilot/utils.ts

```diff
@@ -1,3 +1,52 @@
-import { LogOutputChannel, window } from "vscode";
+import { DocumentSymbol, LogOutputChannel, SymbolKind, TextDocument, commands, window, Range, Selection } from "vscode";
+
+export const CLASS_KINDS: SymbolKind[] = [SymbolKind.Class, SymbolKind.Interface, SymbolKind.Enum];
+export const METHOD_KINDS: SymbolKind[] = [SymbolKind.Method, SymbolKind.Constructor];
 
 export const logger: LogOutputChannel = window.createOutputChannel("Java Rewriting Suggestions", { log: true });
+
+export async function getContainedClassesOfRange(range: Range | Selection, document: TextDocument): Promise<DocumentSymbol[]> {
```

Comment:
It's not easy to understand the name `getContainedClassesOfRange` and `getContainingClassOfRange`.

-> getClassesInRange ?

Reply:
I'll add javadoc to explain these methods.

Reply:
it's better to use the method name to explain itself rather than the doc.

Reply:
`getClassesInRange` can not explain what it does exactly (I added js doc), and lose symmetry with  `getContainerClassOfRange`. I would like to keep `getContainedClassesOfRange` and `getContainerClassOfRange`.

Reply:
renamed as `getClassesContainedInRange` and `getInnermostClassContainsRange` to get some symmetry.